### PR TITLE
Add docs to `keyboardEventMatchesKeys` method

### PR DESCRIPTION
**Pull request by jakemac53**

### CODE REVIEW

File: iron-a11y-keys-behavior.html [hunk 1]
```diff
@@ -310,6 +310,13 @@
         this._resetKeyEventListeners();
       },
 
+      /**
+       * Returns true if a keyboard event matches `eventString`.
+       *
+       * @param {Event} event
```
Comment (jakemac53) [inline iron-a11y-keys-behavior.html]: not sure what the proper type for this should be?

Comment (cdata) [inline iron-a11y-keys-behavior.html]: I would use `KeyboardEvent`

**Comment by cdata** [comment]
Thanks for the contribution. This is looking good. One minor change to make.

**Comment by jakemac53** [comment]
Updated

**Comment by valdrinkoshi** [comment]
@jakemac53 could you please rebase with master and squash the commits into a single one?

**Comment by jakemac53** [comment]
done

**Pull request merged**